code/modules/goonhub/api/routes/rounds/UpdateGameRound.dm

```diff
@@ -0,0 +1,9 @@
+
+/// PUT /rounds/{gameRound}
+/// Update a game round. This should be used when game round data we care about is set after the start of the round.
+/datum/apiRoute/rounds/update
+	method = RUSTG_HTTP_METHOD_PUT
+	path = "/rounds"
+	routeParams = list("gameRound")
+	body = list("game_type")
```

Comment:
no, this isn't just a string body. it's a full json with a type, do this properly.